README.md

```diff
@@ -1,2 +1,39 @@
 # kafka-error-consumer
-Consumer for sending messages from a Kafka error topic to a retry topic
+
+## Introduction
+
+A generic consumer application to move messages from a Kafka error topic to a retry topic.
+
+This application will move records from its configured error topic into the configured retry topic.
+
+The default behaviour is to start at the previous offset and continue processing until it reaches the offset which was current when the application started. The application then stops consuming records, but does not end so that it is not restarted by the infrastructure.
```

Comment:
Which behaviour is default for a new error consumer?
1. Start at the first offset in the topic
2. Start at the next write offset

Reply:
New consumer starts at the first offset - updated README. This si configurable in the KafkaConsumerConfig although I haven't externalized that config.
e.g.
        props.put(
                ConsumerConfig.AUTO_OFFSET_RESET_CONFIG,
                "earliest");